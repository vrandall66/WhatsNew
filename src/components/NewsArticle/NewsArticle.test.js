import React from "react";
import { shallow } from "enzyme";
import { FaArrowRight } from "react-icons/fa";

describe("NewsArticle", () => {
  it("should match the snapshot", () => {
    let img, headline, description, url;
    const wrapper = shallow(
      <div className="NewsArticle">
        <img src={img} alt="" className="NewsArticle-img" />
        <h2>{headline}</h2>
        <p>{description}</p>
        <a href={url} className="NewsArticle-link">
          Link to article
          <span>
            <FaArrowRight />
          </span>
        </a>
      </div>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
