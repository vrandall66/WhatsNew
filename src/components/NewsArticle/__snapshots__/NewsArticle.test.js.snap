// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewsArticle should match the snapshot 1`] = `
<div
  className="NewsArticle"
>
  <img
    alt=""
    className="NewsArticle-img"
  />
  <h2 />
  <p />
  <a
    className="NewsArticle-link"
  >
    Link to article
    <span>
      <FaArrowRight />
    </span>
  </a>
</div>
`;
